apps/storefront-e2e/src/support/page-fragments/totals.fragment.ts

```diff
@@ -1,5 +1,11 @@
-export class CartTotalsFragment {
-  getWrapper = () => cy.get('oryx-cart-totals');
+export class TotalsFragment {
```

Comment:
This fragment is used in 3 different places, but it is different everywhere 

https://github.com/spryker/oryx/pull/698/files#diff-c53e5dd346244eee79cbbfee128a66db8bdecfda4a019cc634fd92f6376d126aR8

https://github.com/spryker/oryx/pull/698/files#diff-6c3c0d4b06a47339eda5a1c49499ad2caa8f6f4c935395210e43a098189c6773R38

https://github.com/spryker/oryx/pull/698/files#diff-86856059ec914c9e7a08d9dd3bf61fdab5fd7d9f4f7f3552c99bc9ff67426162R14

Can we change it to make it consistent?

Reply:
The wrapper is different (`oryx-order-totals` and `oryx-cart-totals`). That’s why rename the fragment and added a constructor param (wrapper: string)

Reply:
class is fine, I mean usage:

```
// private member
private cartTotals = new TotalsFragment('oryx-cart-totals');
```

```
// getter
getCartTotals = new TotalsFragment('oryx-cart-totals');
```

```
// public member with initialization in constructor
this.orderTotalsFragment = new TotalsFragment('oryx-order-totals');
```

we can make it `getter` everywhere